bin/methods/install.js

```diff
@@ -0,0 +1,22 @@
+const cwd = process.cwd()
+const log = require('log-cb')
+const npm = require('npm')
```

Comment:
but maybe you're installing other scripts too. I want to have it automatically added without touching any configuration or writing the names static into a file.

just `cordlr install whatever` and restart the bot. it works already, and I used the plugins folder so we don't get stuff mixed up with usual node_modules.